contracts/validators/CredentialAtomicQueryMTPValidator.sol

```diff
@@ -21,7 +21,7 @@ contract CredentialAtomicQueryMTPValidator is CredentialAtomicQueryValidator {
         uint256[] memory params = new uint256[](5);
         params[0] = inputs[2]; // queryHash
         params[1] = inputs[5]; // gistRoot
-        params[2] = inputs[6]; // issuerId
+        params[2] = inputs[6]; // identityId
```

Comment:
For validator smart contracts it's still issuerId. Please roll these changes back.